Index page (#14)
Luos-io/doc: src/index.md

**Today, Luos is exactly like a <a href="https://en.wikipedia.org/wiki/Microservices" target="_blank">microservices architecture</a>: it encapsulates any software or hardware function to make it communicate and work with any other encapsulated module, however it was developed.**

We have imagined Luos around three key concepts:
 * **Luos Core:** a tiny library that can be added to the MCU of an embedded board to make it seamlessly communicate with other boards. When conceiving electronic devices, you should be able to compose it from multiple functional parts and not think about communication issues between these parts.
 * **Luos Driver:** a hardware abstraction to make the integration of a sensor, an effector or a function as easy at it should be &mdash;that is, with one line of code! Drivers must use standardized APIs to let you switch from one type of sensors to another without breaking anything. For example, all types of distance sensors should return a distance in meters using a same function `get_distance()`.
 * **Robus:** A communication bus dedicated to embedded system communication, to make all the above natural and simple. 

> 

Reviewer: I don't really understand this part. This page is for people discovering what we are doing, they don't care about code parts for now.
Instead we should redirect reader to the relevant part for him by asking some questions and give him the best link to start.
```suggestion
**You are not familiar with Luos operation ?**
 ->  Start by reading the [General use](./_pages/first_steps/general-use.md) page.
 
**You want to make your own board and [Luos drivers](./_pages/modules/drivers.md) or [Luos apps](./_pages/modules/apps.md)?**
 -> Start by reading how to integrate Luos in you [Development environment](./_pages/integrating_luos/dev-env.md).
 -> Then consider to [Create and initialize a module](./_pages/modules/create-init.md).

```

**Luos works like <a href="https://en.wikipedia.org/wiki/Microservices" target="_blank">microservices architecture</a> in the software world, and a <a href="https://en.wikipedia.org/wiki/Distributed_operating_system" target="_blank">distributed operating systems</a>: it encapsulates any software or hardware function to make it communicate and work with any other encapsulated module, however it was developed, either on bare metal or on top of an embedded OS.**
Watch this video for additional details:

<iframe width="800" height="450" src="https://www.youtube.com/watch?v=xQe3z0M_FE8?feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br />

**You are not familiar with Luos operation?**
 ->  Start reading the [General use](./_pages/first_steps/general-use.md) page.
 
**You want to make your own board with [Luos drivers](./_pages/modules/drivers.md) or [Luos apps](./_pages/modules/apps.md)?**
 -> Start reading how to integrate Luos in you [Development environment](./_pages/integrating_luos/dev-env.md).
 -> Then learn how to [Create and initialize a module](./_pages/modules/create-init.md).

> 

Reviewer: ```suggestion
**You want to use [pre-maid board](!!!!There is no such a page Planned!!!!!) and [Luos drivers](./_pages/modules/drivers.md) ?**
 -> You can make your own embedded [Luos apps](./_pages/modules/apps.md).
 -> You can control your devices trough a [Gate](./_pages/modules/modules_list/gate.md) modules using [Pyluos](./_pages/first_steps/pyluos.md).
```

Reviewer: whatch out in Summary.md **Boards list** is linked to **Button** page. We could make a specific page listing boards. In my suggestion I can't find such a page so I put a lot of !!!!! instead...

<hr>

If the Luos technology is something new for you, the following pages will allow you to understand basic concepts.

> 

Reviewer: ```suggestion
```

Watch this video for additional details:

<iframe width="800" height="450" src="https://www.youtube.com/embed/xQe3z0M_FE8?feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br />

#### You are not familiar with Luos operation?

&#8594; Start reading the [General use](./_pages/first_steps/general-use.md) page.
 
#### You want to make your own board with [Luos drivers](./_pages/modules/drivers.md) or [Luos apps](./_pages/modules/apps.md)?

&#8594; Start reading how to integrate Luos in you [Development environment](./_pages/integrating_luos/dev-env.md).
 
&#8594; Then learn how to [Create and initialize a module](./_pages/modules/create-init.md).
 

> #### You want to use [Prototyping boards](./_pages/prototyping_boards/boards-list.md) and [Luos drivers](./_pages/modules/drivers.md)?

Reviewer: ```suggestion
#### You simply want to shape your device behavior?

&#8594; Luos provide a sets of [Prototyping boards](./_pages/prototyping_boards/boards-list.md) you can use as example or to develop your project.
```

&#8594; You can make your own embedded [Luos apps](./_pages/modules/apps.md).
 
&#8594; You can control your devices through a [Gate](./_pages/modules/modules_list/gate.md) module using [Pyluos](./_pages/first_steps/pyluos.md).

>  

Reviewer 2: ```suggestion
 
 If you have questions about a specific topic, you can refer or ask it on the [Luos' Forum](https://community.luos.io). And if you have suggestions about this documentation don't hesitate to create pull requests.
 
```
